msgraph-developer-proxy/responses.json

```diff
@@ -1,104 +1,3 @@
-        "officeLocation": "12/1110",
-        "preferredLanguage": "en-US",
-        "surname": "Bowen",
-        "userPrincipalName": "MeganB@M365x214355.onmicrosoft.com",
-        "id": "48d31887-5fad-4d73-a9f5-3c356e68a038"
-      }
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/me",
-      "responseBody": {
-        "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity",
-        "businessPhones": [
-          "+1 412 555 0109"
-        ],
-        "displayName": "Megan Bowen",
-        "givenName": "Megan",
-        "jobTitle": "Auditor",
-        "mail": "MeganB@M365x214355.onmicrosoft.com",
-        "mobilePhone": null,
-        "officeLocation": "12/1110",
-        "preferredLanguage": "en-US",
-        "surname": "Bowen",
-        "userPrincipalName": "MeganB@M365x214355.onmicrosoft.com",
-        "id": "48d31887-5fad-4d73-a9f5-3c356e68a038"
-      }
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/me",
-      "method": "PATCH",
-      "responseBody": {},
-      "responseCode": 204
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/users/48d31887-5fad-4d73-a9f5-3c356e68a038",
-      "responseBody": {
-        "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity",
-        "businessPhones": [
-          "+1 412 555 0109"
-        ],
-        "displayName": "Megan Bowen",
-        "givenName": "Megan",
-        "jobTitle": "Auditor",
-        "mail": "MeganB@M365x214355.onmicrosoft.com",
-        "mobilePhone": null,
-        "officeLocation": "12/1110",
-        "preferredLanguage": "en-US",
-        "surname": "Bowen",
-        "userPrincipalName": "MeganB@M365x214355.onmicrosoft.com",
-        "id": "48d31887-5fad-4d73-a9f5-3c356e68a038"
-      }
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/me/photo",
-      "responseCode": 404
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/me/photo/$value",
-      "responseBody": "@picture.jpg",
-      "responseHeaders": {
-        "content-type": "image/jpeg"
-      }
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/users/*/photo/$value",
-      "responseBody": "@picture.jpg",
-      "responseHeaders": {
-        "content-type": "image/jpeg"
-      }
-    },
-    {
-      "url": "https://graph.microsoft.com/v1.0/users/*",
-      "responseBody": {
-        "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#users/$entity",
-        "businessPhones": [
-          "+1 425 555 0109"
-        ],
-        "displayName": "Adele Vance",
-        "givenName": "Adele",
-        "jobTitle": "Product Marketing Manager",
-        "mail": "AdeleV@M365x214355.onmicrosoft.com",
-        "mobilePhone": null,
-        "officeLocation": "18/2111",
-        "preferredLanguage": "en-US",
-        "surname": "Vance",
-        "userPrincipalName": "AdeleV@M365x214355.onmicrosoft.com",
-        "id": "87d349ed-44d7-43e1-9a83-5f2406dee5bd"
-      }
-    }
-  ]
+  "responses": []
```

Comment:
Nice! I like this way of giving our users an empty and ready to use set of responses.
Should we output the sample.responses.json file to the output directory so that it's in the zip file users get when they download a release?

Reply:
Great idea! That way they can copy-paste from us having a starting point 👍